src/DistributedLock.Tests/AbstractTestCases/Data/ConnectionStringStrategyTestCases.cs

```diff
@@ -110,7 +110,7 @@ public async Task TestAccessingHandleLostTokenWhileKeepaliveActiveDoesNotBlock()
                     handle.HandleLostToken.Register(() => { });
                 }
             });
-            Assert.IsTrue(await accessHandleLostTokenTask.TryWaitAsync(TimeSpan.FromSeconds(5)));
+            Assert.That(await accessHandleLostTokenTask.TryWaitAsync(TimeSpan.FromSeconds(5)), Is.True);
```

Comment:
Drawing attention here.
```suggestion
            Assert.That(await accessHandleLostTokenTask.TryWaitAsync(TimeSpan.FromSeconds(5)));
```
is also valid and is called the condensed form. For now, I went with the more verbose yet clearer form, but in some cases (e.g. https://github.com/madelson/DistributedLock/pull/203/files#diff-da270224eb19c911add476281ecc4508595b322d7a34c53254e4a82bcaee6e50R24), removing `, Is.True` can be more readable.